facebook_bot.js

```diff
@@ -99,10 +100,18 @@ if(ops.lt === false && ops.ltsubdomain !== null) {
     process.exit();
 }
 
+var access_token = process.env.page_token;
+try {
+    var access_token = JSON.parse(access_token);
```

Comment:
you are right. it is indeed not the most common way. i was hoping to make the minimum changes possible and make it backward compatible... 

in our app, we saved this info in database... but for sample purpose, do you prefer to use a json file instead if passing in json as a env variable really bothers you?